includes/class-newspack-ads-gam.php

```diff
@@ -495,16 +495,37 @@ function( $key ) {
 		return [];
 	}
 
+	/**
+	 * Verify WP environment to make sure it's safe to use GAM.
+	 * 
+	 * @return bool Wether it's safe to use GAM.
+	 */
+	public static function is_sdk_compatible() {
+		// Constant Contact Form plugin loads an old version of Guzzle that breaks the SDK.
+		if ( class_exists( 'Constant_Contact' ) ) {
+			return false;
+		}
+		return true;
+	}
+
 	/**
 	 * Get GAM connection status.
 	 *
 	 * @return object Object with status information.
 	 */
 	public static function connection_status() {
+		if ( false === self::is_sdk_compatible() ) {
+			return [
+				'incompatible' => true,
+				'can_connect'  => false,
+				'connected'    => false,
+				'error'        => __( 'Cannot connect to Google Ad Manager.', 'newspack-ads' ),
```

Comment:
It'd be nice to state the reason here 
```suggestion
				'error'        => __( 'Cannot connect to Google Ad Manager. This WordPress instance is not compatible with this feature.', 'newspack-ads' ),
```